.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
We should set this file as ignore. Each user can define their own vscode settings.

Reply:
@ryandotfurrer how are you deciding what goes in a follow-up vs. is done right now?

I totally understand the need to push things out to another PR but this and the above seem fairly straightforward?

Reply:
Also the below lol

Reply:
@mhchen Since both were incredibly straight-forward, I did both locally and will push the changes. 

I forget that I can do changes inside GitHub itself.

Reply:
@mhchen @shashilo Disregard my comment about "next PR." I meant to say "Next commit"

Reply:
Fixed in latest commit.

Reply:
Aha got it! Well I wrote this anyway so I might as well post it:

> Doesn't matter how you do it, I mean in general, do you do it now and then ask for another review, or do you push it out to a follow-up?

> Factors that make for a good candidate to push to another PR:
* Feedback requires a substantial change + design thinking, re-testing of pre-existing functionality
* PR has gone on for a long time
* PR is blocking your future work/other people's work

> I kind of think it needs to be all 3 to qualify for pushing to another PR

Reply:
Ty Mike! I appreciate it. Foolish mistake by me.
_pours another cup of coffee in an attempt to wake up_